packages/client/hmi-client/src/page/data-explorer/DataExplorer.vue

```diff
@@ -161,16 +169,18 @@ const isLoading = ref<boolean>(false);
 const dirtyResults = ref<{ [resourceType: string]: boolean }>({});
 
 const clientFilters = computed(() => queryStore.clientFilters);
-const xddDataset = computed(() =>
-	resourceType.value === ResourceType.XDD ? resources.xddDataset : 'Terarium'
-);
 
 const assetOptions = ref([
 	{ label: 'Documents', value: ResourceType.XDD, icon: 'pi pi-file' },
 	{ label: 'Models', value: ResourceType.MODEL, icon: 'pi pi-share-alt' },
 	{ label: 'Datasets', value: ResourceType.DATASET, icon: 'pi pi-database' }
 ]);
 
+const topicOptions = ref([
+	{ label: 'Covid-19', value: 'xdd-covid-19' },
+	{ label: 'Climate Change', value: 'climate-change-modeling' }
```

Comment:
```suggestion
	{ label: 'Climate Weather', value: 'climate-change-modeling' }
```

Sorry, because it is an U.S.A. project, we cannot use the words _Climate Change_ in the app.

Reply:
fair enough!